modules/ROOT/pages/enable-and-disable-a-public-portal.adoc

```diff
@@ -0,0 +1,76 @@
+= Enable and Disable a Public Portal
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+All public portals in Exchange created after July 2022 are disabled by default. You cannot sign in to view assets or share new assets on your public portal unless you enable them. You must be an Exchange administrator to enable or disable a public portal.
+
+To determine of your public portal is enabled or disabled, use the following request:
+
+----
+curl "https://anypoint.mulesoft.com/exchange/api/v2/portals/YOUR_ORGANIZATION_DOMAIN/status"
+----
+
+
+== Enable a Public Portal
+
+All assets that are shared in the public portal do not require authorization to view the details of an asset or download the specifications.
```

Comment:
```suggestion
You do not require authorization to view the details or download the specifications of assets shared in the public portal.
```

Reply:
the sentence was just a little wonky toward the end so trying to revise it, maybe could also write

Users do not require authorization to....